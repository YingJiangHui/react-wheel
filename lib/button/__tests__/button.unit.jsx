import Button from '../button'
import renderer from 'react-test-renderer'
import React from 'react'

describe('测试按钮',()=>{
  it('按钮快照',()=>{
    const json = renderer.create(<Button/>).toJSON()
    expect(json).toMatchSnapshot()
  })
})