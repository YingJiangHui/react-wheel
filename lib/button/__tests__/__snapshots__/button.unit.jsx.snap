// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`测试按钮 按钮快照 1`] = `
<div>
  按钮
</div>
`;
